docs/CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Added
+-----
+* Add ``prepare-geo-chipseq`` and ``prepare-geo-rnaseq`` processes
```

Comment:
Write a more informative note, what are this processes for? We have started using the change log to inform our users of the changes with each release so this has become more important.